elixir/plan.sh

```diff
@@ -18,5 +19,7 @@ do_prepare() {
 
 do_build() {
     fix_interpreter "rebar" core/coreutils bin/env
+    fix_interpreter "rebar3" core/coreutils bin/env
```

Comment:
These lines "feel" wrong. It's fixing a file that it doesn't expressly own to point at the env path to then load whichever binary it finds in the current PATH.
Shouldn't they either point directly to the rebar path itself, or should this be part of the erlang plan?